app/models/display_ad.rb

```diff
@@ -173,6 +173,10 @@ def exclude_article_ids=(input)
     write_attribute :exclude_article_ids, (adjusted_input || [])
   end
 
+  def comma_separated_geolocations
+    target_geolocations.map { |g| "#{g.to_iso3166}, " }.join.chomp(", ")
```

Comment:
I _think_ you can accomplish same with `.map(&:to_iso3166).join(", ")`?

Reply:
It seems like this code belongs more to a helper than to a model :eyes: 
(as it's used to format data in the view)